src/components/ModalLedgerSignToken.js

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState, useEffect} from 'react';
+import $ from 'jquery';
+import ledger, { LedgerError, serializeTokenInfo } from '../utils/ledger';
+import tokens from '../utils/tokens';
+import { Buffer } from 'buffer';
+import { IPC_RENDERER } from '../constants';
+import { t } from 'ttag';
+import LedgerSignTokenInfo from './LedgerSignTokenInfo';
+
+/**
+ * Component that shows a modal to add one specific unknown token to the wallet
+ *
+ * @memberof Components
+ */
+function ModalLedgerSignToken({token, modalId}) {
+  const [showClose, canClose] = useState(true);
```

Comment:
This is not being used

Reply:
It was replaced by the `[onLedger, blockClose]` since it's more explicit what is means, but I forgot the cleanup, i'm removing this.